src/main/java/vazkii/quark/world/gen/RealisticGenScreen.java

```diff
@@ -0,0 +1,19 @@
+package vazkii.quark.world.gen;
+
+import net.minecraft.client.gui.screen.BiomeGeneratorTypeScreens;
+import net.minecraft.world.biome.provider.OverworldBiomeProvider;
+import net.minecraft.world.gen.ChunkGenerator;
+import net.minecraft.world.gen.DimensionSettings;
+
+public class RealisticGenScreen extends BiomeGeneratorTypeScreens {
```

Comment:
Is this a client side only class? If so, it should be in /world/client

Reply:
Yeah, it is a client side class. I'll move it.